CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## XX.Y.Z
```

Comment:
this is just a placeholder for when we make the next release?

Reply:
Yeah. I think it's better for us to update the CHANGELOG during the release process, to avoid conflicts while merging PRs, but sometimes we do it during the PR.